Conch/t/integration/05_v1_schema.t

```diff
@@ -31,10 +31,18 @@ my $uuid = Data::UUID->new;
 my $pgtmp = mk_tmp_db() or BAIL_OUT("failed to create test database");
 my $dbh = DBI->connect( $pgtmp->dsn );
 
+my $test_validation_plan = {
+	name        => 'Conch v1 Legacy Plan: Server',
+	description => 'Test Plan',
+	validations => [ { name => 'product_name', version => 1 } ]
+};
+
 my $t = Test::MojoSchema->new(
 	Conch => {
 		pg      => $pgtmp->uri,
-		secrets => ["********"]
+		secrets => ["********"],
+		features => { new_validation => 1 },
```

Comment:
Is there actually a feature flag for the new validation system? There's one in conch.conf.dist but I can't find where it is used. I think it'd probably be a good idea to feature flag it, if we can, so that we can switch to the old system without a full rollback.

Reply:
It is. It's used within the DeviceReport controller. See https://github.com/joyent/conch/blob/master/Conch/lib/Conch/Controller/DeviceReport.pm#L85-L90